keras_cv/layers/preprocessing/base_image_augmentation_layer.py

```diff
@@ -107,6 +105,24 @@ def augment_image(self, image, transformation):
     def __init__(self, seed=None, **kwargs):
         super().__init__(seed=seed, **kwargs)
 
+    @property
+    def force_output_ragged_images(self):
+        """Control whether to force outputting of ragged images."""
+        return getattr(self, "_force_output_ragged_images", False)
```

Comment:
What's the tradeoff between using these settable properties vs a constructor argument?

I feel like `force_ragged=True` or similar could be a nice construction-time control of this

Reply:
So the reason you don't want to do this as a constructor argument is that there is NO user use case to tweak this argument.

A given layer ALWAYS has this set, or not set.  That's why we made `auto_vectorize` an attribute as well.  Basically the goal is to fully hide this from users.

Reply:
+1 -- how does user know which layers should force ragged output, and which should be false?